io/src/main/java/org/commonjava/maven/ext/manip/rest/DefaultVersionTranslator.java

```diff
@@ -19,116 +19,172 @@
 import com.mashape.unirest.http.Unirest;
 import com.mashape.unirest.http.exceptions.UnirestException;
 import org.apache.commons.codec.binary.Base32;
-import org.apache.commons.lang.math.RandomUtils;
 import org.commonjava.maven.atlas.ident.ref.ProjectVersionRef;
-import org.commonjava.maven.ext.manip.ListUtils;
-import org.commonjava.maven.ext.manip.rest.exception.ClientException;
 import org.commonjava.maven.ext.manip.rest.exception.RestException;
-import org.commonjava.maven.ext.manip.rest.exception.ServerException;
 import org.commonjava.maven.ext.manip.rest.mapper.ProjectVersionRefMapper;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.slf4j.MDC;
 
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Random;
+import java.util.*;
 
 import static org.apache.commons.lang.StringUtils.isNotEmpty;
 
 /**
  * @author vdedik@redhat.com
+ * @author Jakub Senko <jsenko@redhat.com>
  */
 public class DefaultVersionTranslator
-    implements VersionTranslator
-{
+        implements VersionTranslator {
     private static final Random RANDOM = new Random();
```

Comment:
There is an existing code style which should be importable into current IDEs. I think it would be preferable to maintain the same codestyle as the rest of the codebase.